src/application-development/async.md

```diff
@@ -0,0 +1,31 @@
+# Async
+
+This section does not serve as an `async` tutorial or teaching material. For these purposes visit [official async-book]. 
```

Comment:
```suggestion
This section does not serve as an async tutorial or teaching material. For these purposes, visit [official async-book]. 
```
Not sure why async is in a code block here, probably not necessary (or desired).